src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -39,6 +39,8 @@ public FindCommand(NameContainsKeywordsPredicate personPredicate,
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
+        assert(personPredicate instanceof NameContainsKeywordsPredicate);
```

Comment:
Good point here, should we all add in similar assertions in our own parts? @AY2223S1-CS2103T-W12-1/developers